addons/html_builder/static/src/plugins/image/image_shape_option_plugin.js

```diff
@@ -143,6 +148,8 @@ class ImageShapeOptionPlugin extends Plugin {
         return shapeSvgText;
     }
     async loadShape(img, newData = {}) {
+        Object.assign(newData, await loadImageInfo(img));
```

Comment:
Why do you need to `loadImageInfo` here? It should already done in `processImage` when necessary.